pkg/cli/util.go

```diff
@@ -36,3 +39,18 @@ func ReadContent(path string) (string, error) {
 	}
 	return string(content), nil
 }
+
+type extraConfig []types.BaseOptionValue
```

Comment:
This type implements [flag.Value](https://golang.org/pkg/flag/#Value), which is useful as the `govc vm.change -e` flag.  In vcpctl, the code can just use `types.OptionValue` directly.

Reply:
Yes, but no need for `types.AnyType()`.  And extra config is always string values, so just `"1"` is fine.